### PR TITLE
Add BitcoinIndonesia

### PR DESCRIPTION
**Pull request by tmlee**

### CODE REVIEW

File: lib/cryptoexchange/exchanges/bitcoin_indonesia/services/market.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          ticker = BitcoinIndonesia::Models::Ticker.new
+          ticker.base = market_pair.base
+          ticker.target = market_pair.target
+          ticker.market = BitcoinIndonesia::Market::NAME
+          ticker.bid = output['buy']
+          ticker.ask = output['sell']
+          ticker.last = output['last']
+          ticker.high = output['high']
+          ticker.low = output['low']
+          ticker.volume = output["vol_#{ticker.base.downcase}"].to_f
```
Comment (tmlee) [inline lib/cryptoexchange/exchanges/bitcoin_indonesia/services/market.rb]: Actually maybe BigDecimal would be safer and sensible

File: lib/cryptoexchange/exchanges/bitcoin_indonesia/bitcoin_indonesia.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+:pairs:
```
Comment (dominicwong617) [inline lib/cryptoexchange/exchanges/bitcoin_indonesia/bitcoin_indonesia.yml]: @tmlee not that it matters, but do we need the extra colon in front? less keystrokes bro....

File: lib/cryptoexchange/exchanges/bitcoin_indonesia/services/market.rb (hunk 1 above)
Comment (dominicwong617) [inline lib/cryptoexchange/exchanges/bitcoin_indonesia/services/market.rb]: @tmlee yes let's use BigDecimal for all exclusively

Comment (tmlee) [inline lib/cryptoexchange/exchanges/bitcoin_indonesia/services/market.rb]: Noted lets create a separate pull request for that

File: lib/cryptoexchange/exchanges/bitcoin_indonesia/bitcoin_indonesia.yml (hunk 2 above)
Comment (tmlee) [inline lib/cryptoexchange/exchanges/bitcoin_indonesia/bitcoin_indonesia.yml]: I want to access the keys using `:`, any better ideas? Write custom methods? The helper methods are only available in Rails package i think

**Pull request merged**